etc/config/analysis.amazon.properties

```diff
@@ -0,0 +1,14 @@
+compilers=&llvm-mca
+compilerType=analysis
+supportsBinary=false
+objdumper=
+demangler=
+
+group.llvm-mca.compilers=llvm-mca
+group.llvm-mca.versionFlag=--version
+
+compiler.llvm-mca.name=llvm-mca (trunk)
+compiler.llvm-mca.exe=/opt/compiler-explorer/clang-trunk/bin/clang++
+compiler.llvm-mca.supportsBinary=false
+# Sets both input and output to Intel variant
+compiler.llvm-mca.intelAsm=--x86-asm-syntax=intel -output-asm-variant=1
```

Comment:
You mention you want to run the compiler with a default value. The only currently available way to do it does not properly show on the UI, but by adding a `.options` key, you will force CE to add whatever you define in there to the compiler invocation options

Reply:
I did try that. But llvm-mca doesn't have the "last argument of a type wins" semantics like clang/gcc have. You can't pass "-mcpu=haswell -mcpu=btver2" and get the btver2 analysis. On clang you'd get code for the last passed -mcpu argument. On llvm-mca, you get an error because you passed that option twice.
Let's worry about the "options" one later. Part of it can also be a discussion about llvm-mca user interface.